tests/monotouch-test/UIKit/UIContentSizeCategoryTest.cs

```diff
@@ -45,6 +45,17 @@ public void Compare ()
 			Assert.Throws<ArgumentException> (() => UIContentSizeCategoryExtensions.Compare (small, (UIContentSizeCategory)271828));
 			Assert.Throws<ArgumentException> (() => ((UIContentSizeCategory)1234).IsAccessibilityCategory ());
 		}
+
+		[Test]
+		public void GetPreferredContentSizeCategoryTest ()
+		{
+			var sizeNSString = UIApplication.SharedApplication.PreferredContentSizeCategory;
+            var sizeEnum = UIContentSizeCategoryExtensions.GetValue(sizeNSString);
+            var size = UIApplication.SharedApplication.GetPreferredContentSizeCategory();
+            Assert.True (sizeEnum == size, "String");
+            var sizeReverse = size.GetConstant();
```

Comment:
```suggestion
            var sizeReverse = size.GetConstant ();
```
small style change (nit) :)